### PR TITLE
Add suppport for custom postgres image

### PR DESCRIPTION
**Pull request by LinAnt**
Adds support for image tags matching x.x(.x)-<custom image>

### CODE REVIEW

File: pkg/admission/validator.go [hunk 1]
```diff
@@ -132,7 +134,10 @@ func ValidatePostgres(client kubernetes.Interface, extClient kubedbv1alpha1.Kube
 
 	// Check Postgres version validation
 	if !postgresVersions.Has(string(postgres.Spec.Version)) {
-		return fmt.Errorf(`KubeDB doesn't support Postgres version: %s`, string(postgres.Spec.Version))
+		ok := postgresCustomVersion.MatchString(string(postgres.Spec.Version))
```
Comment (tamalsaha) [inline pkg/admission/validator.go]: This check is not enough. This just checks that regex matches. Now, you need to extract the version and compare against the `postgresVersions`

Comment (tamalsaha) [reply pkg/admission/validator.go]: You can use https://play.golang.org/p/vADo02vHjW6 to extract version .

**Comment by LinAnt** [comment]
Glide freezes/ gets stuck when I try to update vendors. Trying to debug it at the moment.

**Comment by alexferreira** [comment]
Hi guys, I need to use timescaledb along with kubedb, searching if there was any way I found this PR, any prediction to have that possibility in kuedb?

**Comment by alexferreira** [comment]
@LinAnt, would you have the dockerfile of this image to send me?

**Comment by LinAnt** [comment]
@alexferreira join the KubeDB slack under kubernetes.slack.com and DM me, I can walk you through the steps. We run a lot of plugins with our postgres, so the docker file is a multistage mess at the moment :)

**Comment by tamalsaha** [comment]
@LinAnt , sorry that I took so much time to review this.

The plan is to implement this feature as described https://github.com/kubedb/project/issues/183#issuecomment-409715800

@annymsMthd is implementing this in #162. Please let us know in the https://github.com/kubedb/project/issues/183 if you have any comments or feedback.

**Pull request closed without merging**